game/src/info/mod.rs

```diff
@@ -682,7 +682,7 @@ fn throughput<F: Fn(&Analytics) -> Vec<(AgentType, Vec<(Time, usize)>)>>(
         LinePlot::new(ctx, series, plot_opts),
     ])
     .padding(10)
-    .bg(app.cs.inner_panel)
```

Comment:
![Screenshot from 2021-02-23 10-55-27](https://user-images.githubusercontent.com/1664407/108893683-38b08900-75c6-11eb-9a4b-23cda0edd946.png)
This became washed out.
Previously:
![Screenshot from 2021-02-23 11-02-05](https://user-images.githubusercontent.com/1664407/108893990-9349e500-75c6-11eb-9b47-eca0fb7a1d70.png)

I'm assuming collapsing `panel_bg` and `section_gb` gets us closer to the next step? Fine with a temporary regression.

Reply:
Hm... yeah this is a weird one. Apparently we have two different ways of styling inner panels.

e.g. the info panel that you posted is one way, the settings pane is the other:

Here's day-theme settings pane from master:
<img width="323" alt="Screen Shot 2021-02-23 at 11 35 25 AM" src="https://user-images.githubusercontent.com/217057/108897806-56ccb800-75cb-11eb-94a0-2c74f874db47.png">

I guess it works for the settings pane because all the text is white, whereas the info panel uses some gray text. 

I think we *do* want to consolidate to a single way of doing inner panels, so for now, I think it's best to prefer the "info panel" style which will temporarily lose the cute section backgrounds in the settings panel, but at least the text will be legible everywhere. 

We'll get the differentiable section backgrounds back when the panel colors are finally applied (after themed text et al).

Reply:
Thanks for the quick fix -- it's really sounding like it's going to be awesome to collapse all of these exceptions.